examples/debug.yaml

```diff
@@ -17,15 +17,15 @@
 #
 
 filters:
-  - reason: "Started|Killing"     # filter events of the specified reason, regular expression like "Killing|Killed" is supported.
+  - reason: ""     # filter events of the specified reason, regular expression like "Killing|Killed" is supported.
     message: ""    # filter events of the specified message, regular expression like "Pulling container.*" is supported.
     minCount: 1    # filter events whose count is >= the specified value.
     type: ""       # filter events of the specified type, regular expression like "Normal|Error" is supported.
     action: ""     # filter events of the specified action, regular expression is supported.
-    kind: "Pod|Service"       # filter events of the specified kind, regular expression like "Pod|Service" is supported.
+    kind: ""       # filter events of the specified kind, regular expression like "Pod|Service" is supported.
     namespace: "default"  # filter events from the specified namespace, regular expression like "default|bookinfo" is supported, empty means all namespaces.
     name: ""       # filter events of the specified involved object name, regular expression like ".*bookinfo.*" is supported.
-    service: "[^\\s]{1,}"  # filter events belonging to services whose name is not empty.
+    service: ""  # filter events belonging to services whose name is not empty.
```

Comment:
> only filter by whether service name is empty or not

I am confused with the title with this change.

Reply:
> > only filter by whether service name is empty or not
> 
> I am confused with the title with this change.

Never mind. This is for debugging, and it logs the events to console.